src/pages/Landing.tsx

```diff
@@ -46,41 +52,43 @@ export const Landing = () => {
   };
 
   return (
-    <div className="Landing__container">
-      <Heading3>Import or generate keypair</Heading3>
+    <Layout.Inset>
+      <div className="Landing__container">
+        <Heading3>Import or generate keypair</Heading3>
 
-      <div className="Landing__buttons">
-        <TextLink
-          onClick={() => setIsConnectAccountModalVisible(true)}
-          variant={TextLink.variant.secondary}
-          disabled={account.status === ActionStatus.PENDING}
-          underline
-        >
-          Provide a secret key (testnet or mainnet)
-        </TextLink>
-
-        <div className="Layout__inline">
+        <div className="Landing__buttons">
           <TextLink
-            onClick={handleCreateAccount}
+            onClick={() => setIsConnectAccountModalVisible(true)}
             variant={TextLink.variant.secondary}
             disabled={account.status === ActionStatus.PENDING}
             underline
           >
-            Generate keypair for new account (testnet only)
+            Provide a secret key (testnet or mainnet)
           </TextLink>
 
-          {!isConnectAccountModalVisible &&
-            account.status === ActionStatus.PENDING && <Loader />}
+          <div className="Layout__inline">
+            <TextLink
+              onClick={handleCreateAccount}
+              variant={TextLink.variant.secondary}
+              disabled={account.status === ActionStatus.PENDING}
+              underline
+            >
+              Generate keypair for new account (testnet only)
+            </TextLink>
+
+            {!isConnectAccountModalVisible &&
+              account.status === ActionStatus.PENDING && <Loader />}
```

Comment:
Think it would be good to use a `isPending` const for `account.status === ActionStatus.PENDING`